client/src/pages/PurchaseCredits/PurchaseCredits.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useContext } from "react";
+import { StripeProvider, Elements } from "react-stripe-elements";
+// import { Grid, Typography, Button, makeStyles, Paper } from "@material-ui/core";
+import { UserContext } from "context/UserContext";
+import axios from "axios";
+import NameForm from "./NameForm";
+// import Cart from "./Cart";
+import SelectAmount from "./SelectAmount";
+// import CheckoutForm from "./CheckoutForm";
+import CardSection from "./CardSection";
```

Comment:
Had to comment this out to test after compilation error. Looks like one of the gitignored files

Reply:
I commented it out for now. Will stop git ignoring in next PR when those components are ready.